content/02.main-text.md

```diff
@@ -21,6 +21,34 @@ Although we requested that some authors participate for their specific expertise
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
 
+### Collaborative writing and reviewing
+
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for more technical writing [@url:https://www.overleaf.com/ @url:https://www.authorea.com/].
+`TODO: survey other existing options, see if there are better references`
+These platforms typically offer version control, multiple permission levels, and other functionality to support multi-author document editing.
+Although they work well for editing, they lack sufficient features for managing a collaborative manuscript and attributing precise credit, which are important for open writing.
+
+We adopted standard software development strategies in order to enable any contributor to edit any part of the manuscript but enforce discussion and review of all proposed changes.
+The GitHub platform provided support for organizing and editing the manuscript.
+We used GitHub _issues_ for organization, opening a new issue for each paper under consideration.
+Within the issue, contributors summarized the research, discussed it (sometimes with the original authors), and assessed its relevance to the review.
+Issues also served as an open to-do list and a forum for debating the main message, themes, and topics of the review.
+
+GitHub and the underlying git version control system also structured the writing process.
+An official master version is _forked_ by individual contributors.
```

Comment:
Does `master` here create confusion with the master branch? The forking is of the repository of record, and forks all the branches (I believe).